src/Apps/W1/External Storage - Document Attachments/app/src/DocumentAttachmentIntegration/DocumentAttachmentExternal.Page.al

```diff
@@ -0,0 +1,204 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.ExternalStorage.DocumentAttachments;
+
+using Microsoft.Foundation.Attachment;
+
+/// <summary>
+/// List page for managing document attachments with external storage information.
+/// Provides actions for upload, download, and deletion operations.
+/// </summary>
+page 8751 "Document Attachment - External"
+{
+    PageType = List;
+    SourceTable = "Document Attachment";
+    Caption = 'Document Attachments - External Storage';
+    UsageCategory = None;
+    ApplicationArea = Basic, Suite;
+    Editable = false;
+    Extensible = false;
+
+    layout
+    {
+        area(Content)
+        {
+            repeater(General)
+            {
+                field("Table ID"; Rec."Table ID")
+                {
+                    ToolTip = 'Specifies the table ID the attachment belongs to.';
+                }
+                field("No."; Rec."No.")
+                {
+                    ToolTip = 'Specifies the record number the attachment belongs to.';
+                }
+                field("File Name"; Rec."File Name")
+                {
+                    ToolTip = 'Specifies the name of the attached file.';
+                }
+                field("File Extension"; Rec."File Extension")
+                {
+                    ToolTip = 'Specifies the file extension of the attached file.';
+                }
+                field("Attached Date"; Rec."Attached Date")
+                {
+                    ToolTip = 'Specifies the date the file was attached.';
+                }
+                field("Attached By"; Rec."Attached By")
+                {
+                    ToolTip = 'Specifies the user who attached the file.';
+                }
+                field("Deleted Internally"; Rec."Deleted Internally")
+                {
+                }
```

Comment:
everything before this control has a Tooltip, everything after has a Caption... this one has neither... ? weird.

Reply:
For the ToolTip
We can keep the ToolTip on the TableExtension for new fields, but the other fields don't have a ToolTip specified on the table level yet.
Therefore the ToolTips needs to be duplicated from the original page in the BaseApp